docs/open-source-guidelines.md

```diff
@@ -137,7 +137,7 @@ Bug reports should contain the following information:
 * Summary: A brief description.
 * Steps to reproduce: How did you encounter the bug? Instructions to reproduce it.
 * Expected behavior: How did you expect it to behave?
-* Actual behavior: How did it actually behave?
+* Actual behavior: How has been its current behavior?
```

Comment:
"How has been its current behavior?" is grammatically incorrect 

what are you trying to say?

`how did it actually behave? ` is grammatically correct